<launch>

    <include file="$(find auv_startup)/launch/common.launch" />
    
    <node pkg="auv_vision" name="gate_locator" type="gate_locator" output="screen">
        <param name="detectorMode" value="vertical" />
        <param name="horizontalToVerticalRelation" value="1.5" />
        <param name="debugVision" value="false" />
    </node>

    <node pkg="auv_pilot" name="top_level_fsm" type="top_level_fsm.py" output="screen">
        <param name="notifyOnFinish" value="true" />
        <param name="mode" value="qualification_simple" />
        <param name="diveDelay" value="20" />
        <param name="initialDepth" value="40" />
        <param name="imuReset" value="true" />
        <param name="qualificationDuration" value="20" /> <!-- March move duration in seconds -->
    </node>

</launch>